code/game/mecha/mecha.dm

```diff
@@ -726,9 +741,27 @@
 		if(O.throwforce)
 
 			var/pass_damage = O.throwforce
+			var/pass_damage_reduc_mod
+			if(pass_damage <= damage_minimum)//Too little to go through.
+				src.occupant_message("<span class='notice'>\The [A] doesn't dents \the [src] paint.</span>")
+				src.visible_message("\The [A] doesn't dents \the [src] armor")

```

Comment:
It might be best to have these messages match the X bounces off the Y's Armor message.

Reply:
I figured it was better to have something distinct, to not have extra confusion. I can change those if you confirm you'd prefer that.

Reply:
'X bouncing off Y' is a better informant than the present, since the message currently already informs attackers that they did no damage.